### DISCUSSION TITLE
听说刮刮乐能刮出金色狗头

### DISCUSSION THREAD

**Discussion by Guessme404** [share121/inter-knot#3241] [其他]
是真的吗
![20240820070154](https://github.com/user-attachments/assets/5c4d7c18-098d-4221-bc25-ab875a4a608c)
<img src="https://bbs-static.miyoushe.com/static/2023/11/16/874ff589832ff66517ac5d2999b75e40_5711211283435347869.png" alt="厨师邦布-啊？" width="75" >

**Comment by biziyvan** [comment]
假的，但是有金色大便

**Reply by ColdLinK7** [reply]
金色答辩是60菲林

**Reply by Fluorine-z** [reply]
我上次刮到金色答辩还是上次

**Comment by FFlaming0** [comment]
金色狗狗没见过，但是有金答辩<img src="https://bbs-static.miyoushe.com/static/2024/01/31/a065264ccf369d43f39162320393d9bf_5251148779871794599.png" alt="花火 我来啦" width="75" >

![金色答辩](https://github.com/user-attachments/assets/58420e69-06ed-4e7d-bf91-7b77f7103c22)

**Comment by lllliufeng** [comment]
你说得对，但是
![有了](https://github.com/user-attachments/assets/7a32de2f-47b8-4314-9cad-6672f8e413cd)

**Comment by meichuanlasi** [comment]
![20240827183609](https://github.com/user-attachments/assets/6a1e6911-0b07-4b6f-8174-88c84c517892)
🌝

**Comment by miheihuyou** [comment]
有过，我记得他冒金光了，60菲林

**Comment by 210flight** [comment]
没见过金狗头，但是看过金便便

**Comment by ghost** [comment]
会刮出金坷垃
